eng/common/scripts/Update-PullRequest-In-ReleasePlan.ps1

```diff
@@ -0,0 +1,74 @@
+param(    
+    [Parameter(Mandatory = $true)]
+    $ReleasePlanWorkItemId,
+    [Parameter(Mandatory = $true)]
+    $PullRequestUrl,
+    [Parameter(Mandatory = $true)]
+    $Status,
+    [Parameter(Mandatory = $true)]
+    $LanguageName,
+    [Parameter(Mandatory = $true)]
+    $AuthToken
```

Comment:
Do we need to pass the authtoken or will we already be in logged in devops session?

Reply:
This is for GitHub auth token to add a comment